cclient/serialization/json/helpers.c

```diff
@@ -302,6 +302,7 @@ retcode_t hash243_stack_to_json_array(hash243_stack_t stack, cJSON* const json_r
       return RC_CCLIENT_FLEX_TRITS;
     }
   }
+
```

Comment:
Is that really the commit purpose?

Reply:
Not at all, this is a conflict residual, I'll remove

Reply:
@tsvisabo Removed